rskj-core/src/main/java/co/rsk/net/sync/SnapSyncState.java

```diff
@@ -247,6 +284,14 @@ public Queue<ChunkTask> getChunkTaskQueue() {
         return chunkTaskQueue;
     }
 
+    public int getBlockHeaderChunkSize() {
+        return syncConfiguration.getChunkSize();
+    }
+
+    public boolean isRunning() {
+        return isRunning == Boolean.TRUE;
+    }
+
```

Comment:
suggestion:

I saw in a lot of places we using the denial of this method: `!state.isRunning()`. Maybe we could add this method here to avoid the `!` repeated times. Something like:

```
    public boolean isNotRunning() {
        return isRunning == Boolean.FALSE;
    }
```

Reply:
I'd prefer the first version, as being more natural and corresponding to the `isRunning` field. Also there's a 3rd state for not running- `null`